rules/symfony4/tests/Rector/ClassMethod/ConsoleExecuteReturnIntRector/Fixture/demo_command.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+namespace Rector\Symfony4\Tests\Rector\ClassMethod\ConsoleExecuteReturnIntRector\Fixture;
+
+final class DemoCommand extends \Symfony\Component\Console\Command\Command
```

Comment:
fixture with no different between `-----` should only written once above only, and use `skip_` prefix, for example:

https://github.com/rectorphp/rector/blob/bf94f4c420ee577403edf1da3ee445830462695d/rules/symfony4/tests/Rector/ClassMethod/ConsoleExecuteReturnIntRector/Fixture/skip_already_int.php.inc